maskrcnn_benchmark/modeling/roi_heads/mask_head/inference.py

```diff
@@ -44,11 +60,13 @@ def forward(self, x, boxes):
         index = torch.arange(num_masks, device=labels.device)
         mask_prob = mask_prob[index, labels][:, None]
 
+        boxes_per_image = [len(box) for box in boxes]
+        
         if self.masker:
+            mask_prob = split_list(mask_prob, boxes_per_image)
```

Comment:
I'm confused: isn't `mask_prob` a single tensor at this stage? If yes, why can't we use `.split` here?